pkg/volume/azure_dd/mounter.go

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
the file name is a misnomer, there is also unmounter there. Keep azure_dd.go and unit tests.